test/multiverse/suites/hybrid_agent/parsing_helpers.rb

```diff
@@ -157,7 +157,8 @@ def verify_span_output(output)
       end
 
       expected['attributes']&.each do |expected_key, expected_value|
-        assert_equal expected_value, actual[1] & [expected_key],
```

Comment:
the `assert_equal` method didn't like the `&`. I hope the spirit of the test is the same, but happy to rework if not.

Reply:
whoops, missed this! does the `&` work in the variable assignment?

Reply:
I dont think I've seen `&` used in this way before - what were we trying to do?

Reply:
Me neither -- it was a Rubocop suggestion. The goal is to make sure that `actual[1]` isn't `nil`. However, maybe that's getting taken care of with the `assert actual` at the top of the method. If a span is found, it's pretty safe that it will have that hash in the first index. 

After thinking about it more, I'm fine with leaving it out! Thanks for asking.